loadtest/user/userentity/user.go

```diff
@@ -199,6 +201,56 @@ func (ue *UserEntity) IsTeamAdmin() (bool, error) {
 	return user.IsInRole(model.TEAM_ADMIN_ROLE_ID), nil
 }
 
+// PromoteToAdmin promotes the given user to a sysadmin role
+func (ue *UserEntity) PromoteToAdmin(u *UserEntity) error {
+	if err := ue.Login(); err != nil {
+		return err
+	}
+
+	isAdmin, err := ue.IsSysAdmin()
+	if err != nil {
+		return err
+	}
+	if !isAdmin {
```

Comment:
This is here for a sanity check, in the case that this func is called from a non-admin user.